### ISSUE TITLE
{{CONTAINER_IP}} have to be replaced by __CONTAINER_IP__ in yml files

### ISSUE THREAD

**Issue by eravion** [elgalu/docker-selenium#312]
Hello,

in this files : 
docker-compose.yml
docker-compose-tests.yml
docker-compose-scales.yml

{{CONTAINER_IP}} have to be replaced by __CONTAINER_IP__

If not, the node is not using the good network interface to communicate with the hub and the hub is very slow.
I tested on a swarm cluster.

Sorry, I'm not familiar with github to do it by myself ...

thanks.

**Comment by diemol** [comment]
Hi @eravion,

Can you please provide more information? Can you share the commands you are executing and the output you are getting?

**Comment by eravion** [comment]
Hello,

I build the docker from the git (git clone, then docker build).
Then I started 1 hub, 2 firefox and 2 chrome on a swarm cluster with 4 nodes.
The access to the hub web page (http://@IP:4444/grid/console) was very slow. This is a classical problem when the node is not using the good interface to communicate with the hub.
In the case of this docker they have 2 interfaces.
I reproduce the issue on two different docker swarm cluster.

This problem is fixed by replacing  {{CONTAINER_IP}} by __ CONTAINER_IP__ in the files listed in the first post.

Maybe this is linked to the swarm cluster, I did not tested on single machine.
